files/build/scripts/apt-get

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Could you add comments to explain the purpose and usage of this script?
My understanding is that they are hooks based on the vanilla executables. So:
1. What is the new feature?
2. Any new command line arguments?
3. How to call vanilla ones?

Reply:
It should be the old draft script, it will print error message if version control enabled and the version is not set.